src/app/repositories/buckets/dataset_bucket_repository.py

```diff
@@ -21,3 +21,17 @@ def get_dataset_file_as_json(self, filename: str) -> RawDataset:
         RawDatasetWithMetadata: raw dataset from the bucket file as json.
         """
         return self.get_bucket_file_as_json(filename)
+
+    def fetch_first_filename_from_bucket(self) -> str | None:
+        """
+        Fetches the first filename from the bucket.
+
+        Returns:
+        str: filename from the bucket.
+        """
+        blobs = self.bucket.list_blobs()
```

Comment:
Can you try to find if there is a way to retrieve the first file according to the creation time? That is to handle the 2nd issue in spike report